x-pack/plugins/notifications/server/routes/api/v1/notifications/notify.js

```diff
@@ -40,7 +47,10 @@ export function checkForErrors(action, actionId, data) {
 
   if (fields.length !== 0) {
     return {
-      message: `Unable to perform '${action.name}' action due to missing required fields.`,
+      message: i18n.translate('xpack.notifications.notify.unableToPerformActionDueToMissingRequiredFieldsMessage', {
+        defaultMessage: `Unable to perform '{actionName}' action due to missing required fields.`,
```

Comment:
I checked with pseudo locale and got: 

![image](https://user-images.githubusercontent.com/11524637/51534202-3531fd00-1e56-11e9-8c39-67213c4016e0.png)

where labels `Šûƀƀĵéççţ`, `Ɓôððý` are from [getMissingField](https://github.com/elastic/kibana/pull/29039/files#diff-dc81bf6ca9f9250fa25b3a8fbcb5de5dR34) function